auth0/src/main/java/com/auth0/android/provider/CallbackHelper.java

```diff
@@ -47,9 +48,19 @@ public CallbackHelper(@NonNull String packageName) {
      * @return the callback URI.
      */
     public String getCallbackURI(@NonNull String domain) {
-        String uri = String.format(REDIRECT_URI_FORMAT, domain, packageName);
-        Log.v(TAG, "The Callback URI is: " + uri);
-        return uri;
+        HttpUrl url = HttpUrl.parse(domain);
+        if (url == null) {
```

Comment:
what kind of error will the user get when the request is made with a null callback URI?
maybe throw an exception is better? it will not work anyway, and the error will be easier to notice